### PR TITLE
add examples to card component

### PR DESCRIPTION
**Pull request by TahimiLeonBravo**

### CODE REVIEW

File: src/components/Card/index.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import CardBoddy from './cardBody';
 import Footer from './footer';
 import Actions from './actions';
 
+/** Cards are used to apply a container around a
+ * related grouping of information. */
 export default function Card(props) {
```
Comment (reiniergs) [inline src/components/Card/index.js]: change the comment to this format 
```js
/**
 * Cards are used to apply a container around a
 * related grouping of information.
 */
```

File: src/components/Card/Readme.md [hunk 2]
```diff
@@ -1,27 +1,69 @@
-Card Expample 1:
+card default with illustration
 
+    <div className="slds-p-vertical_large slds-p-horizontal_large">
```
Comment (LeandroTorresSicilia) [inline src/components/Card/Readme.md]: you can use here slds-p-around_large instead

File: src/components/Card/Readme.md [hunk 3]
```diff
@@ -1,27 +1,69 @@
...
+        <Card className="slds-card_boundary">
+           <img 
+            src="images/illustrations/empty-state-assistant.svg"
+            className="slds-p-vertical_x-large slds-align_absolute-center"
+            alt="the wood image" />
```
Comment (LeandroTorresSicilia) [inline src/components/Card/Readme.md]: the wood image is redundant only use wood o the wood

File: src/components/Card/Readme.md [hunk 4]
```diff
@@ -1,27 +1,69 @@
...
+
+    <div className="slds-m-vertical_large slds-m-horizontal_large">
+        <Card
+            className="slds-card_boundary"
+            iconName="standard:task"
+            title="Task"
+            actions={<Button variant="neutral" label="New" />} >
+
+                <div className="slds-p-vertical_large slds-align_absolute-center slds-grid_vertical" >
+                    <img src="images/illustrations/empty-state-tasks.svg" alt="the wood image" />
```
Comment (LeandroTorresSicilia) [inline src/components/Card/Readme.md]: same ^^ only wood

**Pull request merged**